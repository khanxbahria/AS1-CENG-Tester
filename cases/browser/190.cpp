#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Browser* brows = new Browser();
Window a;
Window b;
Window c;
Window d;
Tab A = Tab("uuu", "m","t");
Tab B = Tab("sss", "e","jj");
Tab C = Tab("ddd", "h","v");
Tab D = Tab("gg", "pp","eee");
Tab E = Tab("vvv", "u","lll");
Tab F = Tab("mm", "jjj","c");
Tab G = Tab("ppp", "gg","ddd");
Tab H = Tab("qq", "q","zzz");
Tab I = Tab("tt", "iii","g");
Tab J = Tab("zzz", "y","iii");
Tab K = Tab("uuu", "ee","kk");
Tab L = Tab("cc", "uu","dd");
Tab M = Tab("b", "fff","z");
Tab N = Tab("oo", "dd","bb");
Tab O = Tab("c", "nnn","vvv");
Tab P = Tab("zzz", "z","i");
Tab Q = Tab("aa", "ooo","oo");
Tab R = Tab("oo", "qq","r");
Tab S = Tab("v", "aa","h");
Tab T = Tab("n", "b","ddd");
Tab U = Tab("kk", "ccc","pp");
Tab V = Tab("eee", "l","s");
Tab W = Tab("sss", "ff","m");
Tab X = Tab("ppp", "ccc","hhh");
Tab Y = Tab("g", "c","ii");
Tab Z = Tab("v", "k","ttt");
a.changeActiveTabTo(0);
b.changeActiveTabTo(0);
c.newTab(A);
d.changeActiveTabTo(0);
a.newTab(B);
b.moveActiveTabTo(5);
c.moveActiveTabTo(5);
d.newTab(C);
a.moveActiveTabTo(3);
b.changeActiveTabTo(3);
c.newTab(D);
d.newTab(E);
a.newTab(F);
b.newTab(G);
c.newTab(H);
d.changeActiveTabTo(2);
a.moveActiveTabTo(3);
b.newTab(I);
c.moveActiveTabTo(3);
d.changeActiveTabTo(3);
a.moveActiveTabTo(1);
b.moveActiveTabTo(1);
c.newTab(J);
d.newTab(K);
a.changeActiveTabTo(4);
b.newTab(L);
c.newTab(M);
d.moveActiveTabTo(4);
a.changeActiveTabTo(5);
b.moveActiveTabTo(5);
c.changeActiveTabTo(5);
d.moveActiveTabTo(5);
a.moveActiveTabTo(4);
b.moveActiveTabTo(4);
c.changeActiveTabTo(4);
d.newTab(N);
a.moveActiveTabTo(3);
b.moveActiveTabTo(3);
c.moveActiveTabTo(3);
d.changeActiveTabTo(3);
a.moveActiveTabTo(2);
b.changeActiveTabTo(2);
c.changeActiveTabTo(2);
d.newTab(O);
a.changeActiveTabTo(5);
b.changeActiveTabTo(5);
c.changeActiveTabTo(5);
d.changeActiveTabTo(5);
a.newTab(P);
b.newTab(Q);
c.changeActiveTabTo(1);
d.moveActiveTabTo(1);
a.newTab(R);
b.moveActiveTabTo(2);
c.changeActiveTabTo(2);
d.moveActiveTabTo(2);
a.newTab(S);
b.moveActiveTabTo(2);
c.newTab(T);
d.moveActiveTabTo(2);
a.changeActiveTabTo(0);
b.newTab(U);
c.newTab(V);
d.moveActiveTabTo(0);
a.moveActiveTabTo(1);
b.moveActiveTabTo(1);
c.newTab(W);
d.newTab(X);
a.changeActiveTabTo(4);
b.newTab(Y);
c.changeActiveTabTo(4);
d.moveActiveTabTo(4);
a.newTab(Z);
b.changeActiveTabTo(2);
c.moveActiveTabTo(2);
d.moveActiveTabTo(2);
a.changeActiveTabTo(0);
b.changeActiveTabTo(0);
d.changeActiveTabTo(0);
a.changeActiveTabTo(0);
c.moveActiveTabTo(0);
d.changeActiveTabTo(0);
b.moveActiveTabTo(2);
c.changeActiveTabTo(2);
d.changeActiveTabTo(2);
a.changeActiveTabTo(2);
b.moveActiveTabTo(2);
d.changeActiveTabTo(2);
c.moveActiveTabTo(0);
d.moveActiveTabTo(0);
b.moveActiveTabTo(5);
c.moveActiveTabTo(5);
a.changeActiveTabTo(3);
b.moveActiveTabTo(3);
c.changeActiveTabTo(3);
a.moveActiveTabTo(3);
b.changeActiveTabTo(3);
c.changeActiveTabTo(3);
a.changeActiveTabTo(0);
b.moveActiveTabTo(0);
c.moveActiveTabTo(0);
d.changeActiveTabTo(0);
a.changeActiveTabTo(1);
b.moveActiveTabTo(1);
c.changeActiveTabTo(1);
d.moveActiveTabTo(1);
a.changeActiveTabTo(5);
b.moveActiveTabTo(5);
c.moveActiveTabTo(5);
a.moveActiveTabTo(2);
c.moveActiveTabTo(2);
b.moveActiveTabTo(1);
c.moveActiveTabTo(1);
d.moveActiveTabTo(1);
d.moveActiveTabTo(0);
a.changeActiveTabTo(2);
a.changeActiveTabTo(2);
c.moveActiveTabTo(2);
d.moveActiveTabTo(2);
c.moveActiveTabTo(4);
d.changeActiveTabTo(4);
a.changeActiveTabTo(3);
b.moveActiveTabTo(3);
c.changeActiveTabTo(3);
d.moveActiveTabTo(3);
a.changeActiveTabTo(4);
b.changeActiveTabTo(4);
a.changeActiveTabTo(1);
d.moveActiveTabTo(1);
b.moveActiveTabTo(4);
a.changeActiveTabTo(4);
b.moveActiveTabTo(4);
a.changeActiveTabTo(5);
b.moveActiveTabTo(5);
d.changeActiveTabTo(5);
a.moveActiveTabTo(3);
b.moveActiveTabTo(3);
c.changeActiveTabTo(3);
d.moveActiveTabTo(3);
b.changeActiveTabTo(4);
a.changeActiveTabTo(1);
b.moveActiveTabTo(1);
d.moveActiveTabTo(1);
b.changeActiveTabTo(2);
d.changeActiveTabTo(2);
b.moveActiveTabTo(5);
d.changeActiveTabTo(5);
b.changeActiveTabTo(5);
c.moveActiveTabTo(5);
d.changeActiveTabTo(5);
d.changeActiveTabTo(0);
b.moveActiveTabTo(5);
c.changeActiveTabTo(5);
brows->addWindow(a);
brows->addWindow(b);
brows->addWindow(c);
brows->addWindow(d);
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->newWindow();
brows->closeEmptyWindows();
brows->switchToWindow(1);
brows->newWindow();
brows->print();
brows->closeWindow();
brows->getWindow(6);
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->getWindow(3);
brows->mergeAllWindows();
brows->closeWindow();
brows->print();
brows->closeEmptyWindows();
brows->closeWindow();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->switchToWindow(3);
brows->mergeAllWindows();
brows->getWindow(9);
brows->print();
brows->print();
brows->closeEmptyWindows();
brows->print();
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->closeWindow();
brows->switchToWindow(6);
brows->newWindow();
brows->mergeAllWindows();
brows->print();
brows->closeWindow();
brows->switchToWindow(8);
brows->print();
brows->newWindow();
brows->closeEmptyWindows();
brows->print();
brows->switchToWindow(2);
brows->closeWindow();
brows->mergeAllWindows();
brows->getWindow(3);
brows->mergeAllWindows();
brows->closeWindow();
brows->newWindow();
brows->mergeAllWindows();
brows->switchToWindow(2);
brows->print();
brows->mergeAllWindows();
brows->closeWindow();
brows->getWindow(9);
brows->newWindow();
brows->closeEmptyWindows();
brows->switchToWindow(6);
brows->getWindow(0);
brows->switchToWindow(0);
brows->getWindow(7);
brows->mergeAllWindows();
brows->switchToWindow(9);
brows->newWindow();
brows->switchToWindow(3);
brows->closeWindow();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->newWindow();
brows->switchToWindow(6);
brows->closeEmptyWindows();
brows->closeWindow();
brows->mergeAllWindows();
brows->switchToWindow(1);
brows->getWindow(0);
brows->getWindow(0);
brows->getWindow(7);
brows->newWindow();
brows->getWindow(6);
brows->mergeAllWindows();
brows->print();
brows->newWindow();
brows->print();
brows->mergeAllWindows();
brows->print();
brows->mergeAllWindows();
brows->switchToWindow(2);
brows->newWindow();
brows->closeWindow();
brows->newWindow();
brows->closeWindow();
brows->print();
brows->getWindow(4);
brows->print();
brows->getWindow(0);
brows->mergeAllWindows();
brows->getWindow(3);
brows->getWindow(7);
brows->closeWindow();
brows->closeWindow();
brows->switchToWindow(7);
brows->mergeAllWindows();
brows->getWindow(4);
brows->switchToWindow(5);
brows->mergeAllWindows();
brows->switchToWindow(5);
brows->newWindow();
brows->closeWindow();
brows->switchToWindow(8);
brows->closeWindow();
brows->newWindow();
brows->getWindow(4);
brows->getWindow(6);
brows->print();
brows->switchToWindow(3);
brows->mergeAllWindows();
brows->getWindow(0);
delete brows;
}
	return 0;
}