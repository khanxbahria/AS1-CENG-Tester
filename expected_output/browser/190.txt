The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 1
e
sss
jj





k
v
ttt

qq
oo
r

jjj
mm
c

z
zzz
i

aa
v
h

The active tab is tab 5
ccc
kk
pp





iii
tt
g

ooo
aa
oo

gg
ppp
ddd

c
g
ii

uu
cc
dd

The active tab is tab 5
b
n
ddd





pp
gg
eee

l
eee
s

m
uuu
t

ff
sss
m

y
zzz
iii

q
qq
zzz

fff
b
z

The active tab is tab 0
dd
oo
bb

ee
uuu
kk

u
vvv
lll





ccc
ppp
hhh

h
ddd
v

nnn
c
vvv

Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
The browser is empty
The browser is empty
The browser is empty
The active tab is tab 0




The browser is empty
The active tab is tab 0




The active tab is tab 0




The active tab is tab 1








Window is empty
The list is empty.
The active tab is tab 0




The active tab is tab 1








Window is empty
The list is empty.
The active tab is tab 2












Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 2












Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 2












Window is empty
The list is empty.
The active tab is tab 0




